source/drivers/php.txt

```diff
@@ -17,44 +17,56 @@ MongoDB PHP Driver
 Introduction
 ------------
 
-The currently maintained driver is the `mongodb
-<https://pecl.php.net/package/mongodb>`_ extension available from PECL.
+The PHP driver consists of two components, the MongoDB `extension <https://github.com/mongodb/mongo-php-driver>`_
+and `library <https://docs.mongodb.com/php-library/current/>`_.
 
-You might consider using the driver with the complimentary `PHP library
-<https://github.com/mongodb/mongo-php-library>`_, which implements a more
-full-featured API on top of the bare-bones driver.
+The extension provides a low-level API and mainly serves to integrate
+`libmongoc and libbson <https://docs.mongodb.com/ecosystem/drivers/c/>`_ with PHP.
 
-Further information on this architecture may be found 
-in the `PHP.net documentation <http://php.net/manual/en/mongodb.overview.php>`_.
+While it is possible to use the extension alone, users are strongly encouraged
+to use the extension and library together. The library provides a high-level API
+consistent with other MongoDB language drivers.
 
-- `Tutorials <https://www.php.net/manual/en/mongodb.tutorial.php>`__
+- `Tutorials <https://docs.mongodb.com/php-library/current/tutorial>`__
 
-- `Driver Architecture and Internals <https://www.php.net/manual/en/mongodb.architecture.php>`__
+- `Library Architecture <https://www.php.net/manual/en/mongodb.architecture.php>`__
 
-- `API Reference <https://php.net/manual/en/book.mongodb.php>`_
+- API Reference
 
-- `Changelog <https://github.com/mongodb/mongo-php-driver/releases>`__
+  - `Library <https://www.php.net/manual/en/set.mongodb.php>`__
 
-- `Source Code <https://github.com/mongodb/mongo-php-driver>`__
+  - `Extension <https://php.net/manual/en/book.mongodb.php>`__
```

Comment:
This links directly to the API reference classes directly under the `MongoDB\Driver` namespace, but it omits other classes in the extension, such as:

 * https://www.php.net/manual/en/book.bson.php
 * https://www.php.net/manual/en/mongodb.monitoring.php
 * https://www.php.net/manual/en/mongodb.exceptions.php

Unlike the library, we don't have a common deep link for all API reference, so I think it'd be best to just use the main documentation page here instead: https://www.php.net/manual/en/set.mongodb.php. Although the page lists articles first, all classes in the extension are linked directly from that page, too.

And similar to my earlier suggestion, you may want to use https://php.net/mongodb instead, since it's a shorter URL and points to the same entry point for the extension docs.

Reply:
I'm guessing this doesn't relate to the source code link, but rather the Extension API ref. I've updated that link as well as the initial extension link.

Reply:
Correct, I'm referring to:

```
Extension <https://php.net/manual/en/book.mongodb.php>`__
```

....from the new side of the diff.

Reply:
Makes sense, I'll get this change in.